src/styles/base/base.scss

```
@@ -0,0 +1,87 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.page {
+  font-family: Cinzel, sans-serif;
+  font-weight: 400;
+  color: $color-text;
+  line-height: 27px;
+
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0;
+  }
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    opacity: 0;
+    transition: all 0.3s;
+    transform: translateX(-100%);
+    pointer-events: none;
+  }
+
+  &__menu:target {
+    opacity: 1;
+    transform: translateX(0);
+    pointer-events: all;
+  }
+
+  &:has(.page__menu:target) {
+    overflow: hidden;
+  }
+}
+
+.main {
+  background-color: $color-primary;
+
+  &__container {
+    display: grid;
+    row-gap: 100px;
+    padding-block: 100px;
+    padding-inline: 20px;
+    margin-inline: auto;
+    max-width: 280px;
+
+    @media (min-width: $breakpoint-tablet) {
```

Good practice make mixin for every media-queries, f.e.: 
`@mixin onTablet {
  @media (min-width: 768px) {
    @content;
  }
}`
```suggestion
    @include onTablet {
```

You don't need fix it now, but notice for future projects